data-fundamentals/README.md

```diff
@@ -17,6 +18,22 @@ The goal of this section is to briefly describe how to apply the fundamentals to
 - [Observability](#observability)
 - [End to End and Azure Technology Samples](#end-to-end-and-azure-technology-samples)
 
+## Concurrency Control
+
+Your systems should (almost) always leverage some form of concurrency control, to ensure correctness amongst competing requests and to prevent data races. The 2 forms of concurrency control are Pessimistic and Optimistic.
```

Comment:
clarity/readability suggestion: bold **pessimistic** and **optimistic** for attention rather than capitalizing, & repeating bolding below for ease of reading. Remove inconsistent capitalization of these terms.

Reply:
done!